tests/stitchData.spec.js

```diff
@@ -9,7 +9,7 @@ const timeline = require('./fixtures/stitched-timeline');
 describe('Stitching timeline to make final data', function() {
   it('should merge trend query and news data on matching dates', function() {
     const expected = timeline;
-    const actual = stitchTimeline(trends, stories);
+    const actual = stitchTimeline(trends, stories, '"clinton"');
```

Comment:
if you're comparing to live data for this test instead of a static saved version of this `clinton` query, i think this test will fail in a month or whenever the API response changes?